packages/async-flow/src/type-guards.js

```diff
@@ -11,7 +11,10 @@ export const FlowStateShape = M.or(
 
 export const PropertyKeyShape = M.or(M.string(), M.symbol());
 
-export const LogEntryShape = M.or(
+export const START_GENERATION_OP_NAME = 'startGeneration';
```

Comment:
Why is it that this one needs the name indirection but the other ops do not? In general my reaction is that I understand why such indirection adds value to C programs, but not to JS programs. Especially not to JS programs that are already checked by TS types that can include such string constants in the type.

But the main thing is consistency. Do this for either all or none.